specifications/api/swagger.yaml

```
@@ -5888,6 +5923,23 @@ paths:
             schema: *ApiSerialisedTransactionEncoded
       responses: *responsesDecodedTransaction
 
+  /wallets/{walletId}/transactions-submit:
+    post:
+      operationId: submitTransaction
+      tags: ["Transactions New"]
+      summary: Submit
+      description: |
+        <p align="right">status: <strong>stable</strong></p>
+        Submit a transaction that was already created and signed.
```

```suggestion
        Submit a transaction that was already created and signed. Fails for foreign transactions.
        
        If the submission to the local node was successful, the wallet will store the pending transaction and retry until confirmed or expired.
```

Maybe something like this to mention:
1. this endpoint will track the transaction as pending
2. that it will fail if the transaction is foreign (without duplication the definition of foreign)

---

corrected